src/components/core/core.js

```diff
@@ -63,14 +62,30 @@ export default class Core extends UIObject {
     return this.getPlugin('strings') || { t: (key) => key }
   }
 
+  get mediaControl() {
```

Comment:
I think we could document all the public interface of this component since it'll be very useful for players developers.

Reply:
@leandromoreira this "public" interface is only for compatibility purposes, in order to not break existing plugins. It will be removed in a later version, so it's already "deprecated"